maintainers/maintainer-list.nix

```diff
@@ -7203,6 +7203,12 @@
     githubId = 157610;
     name = "Piotr Bogdan";
   };
+  pborzenkov = {
+    email = "pavel@borzenkov.net";
+    github = "pborzenkov";
+    githubId = 434254;
+    name = "Pavel Borzenkov";
+  };
```

Comment:
Please add yourself as a maintainer in a separate commit with the message `maintainers: add pborzenkov`.

Reply:
Done!